### ISSUE TITLE
Could not find module `emberx-file-input/components/x-file-input`

### ISSUE THREAD

**Issue by kevinansfield** [thefrontside/emberx-file-input#57]
I'm not sure what's going on because I can't see anything that would cause a problem in the addon's commit history but when upgrading from `1.1.2` to `1.2.0` the import we have in our component override ceases to work:

```
Uncaught Error: Could not find module `emberx-file-input/components/x-file-input` imported from `ghost-admin/components/gh-file-input`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at Class._extractDefaultExport (index.js:392)
    at Class.resolveOther (index.js:113)
    at Class.superWrapper [as resolveOther] (ember-utils.js:437)
    at Class.resolve (resolver.js:144)
    at resolve (container.js:852)
```

The import in question looks like this:

```
import XFileInput from 'emberx-file-input/components/x-file-input';
```

If I remove our override component and use `{{x-file-input}}` directly I'm still seeing an error:

```
Assertion Failed: A component or helper named "x-file-input" could not be found
```

**Update:** It looks like the package on npm is missing the most important files. The `addon/components` and `vendor` dirs are missing and the `app` dir is empty:

<img width="770" alt="screen shot 2017-11-22 at 12 52 45" src="https://user-images.githubusercontent.com/415/33128375-169262ee-cf84-11e7-8684-0d8b0cd02a56.png">

**Comment by byrnedo** [comment]
Also experiencing this.

**Comment by janfryauf** [comment]
I had to change it to the exact version:
```
"emberx-file-input": "1.1.2"
```

if its ^1.2.0 or ^1.1.2 it's not working because in node_modules the folder emberx-file-input/addon/components is empty

**Comment by cowboyd** [comment]
@kevinansfield Did anything else change except the version?

@janfryauf That is very strange indeed. I wonder if it's a packaging error? Like maybe it is not present in the tarball itself?

**Comment by kevinansfield** [comment]
> @kevinansfield Did anything else change except the version?

No, it looks like a packaging error, most of the files are missing from the tarball on npm.

**Comment by sihuang** [comment]
@kevinansfield did you manage to find a solution?

I'm seeing "Compile Error: x-file-input is not a helper" in browser console.

**Comment by kevinansfield** [comment]
@sihuang I stuck to to 1.1.2. Until the 1.2.0 package is re-published I don't think it's usable as most of the addon files are missing from the tarball. You could always point npm directly to the tag on the repo if you want to go in that direction.

**Comment by Robdel12** [comment]
I'm going to try and take care of this tonight. I'm curious as to _how_ it ended up being published with missing files.

**Comment by Robdel12** [comment]
This should be taken care of with v1.2.1, let me know if there are any more issues!

**Comment by sihuang** [comment]
@Robdel12 thanks a lot. will give it a try first change I get.

**Comment by sihuang** [comment]
@Robdel12 Confirmed. Problem resolved in 1.2.1 and missing files are back. Thanks again and keep up the good work.

**Comment by Robdel12** [comment]
Woo! Thanks for confirming that for me :D
